scripts/test_gsm8k.py

```diff
@@ -0,0 +1,67 @@
+from braintrust import Eval, Score
+from autoevals.value import ExactMatch
+from datasets import load_dataset
+from openai import AsyncOpenAI
+from pydantic import BaseModel
+import instructor
+from asyncio import run
+from uuid import uuid4
+
+dataset = load_dataset("567-labs/gsm8k")
+oai = AsyncOpenAI()
+
+
+class Answer(BaseModel):
+    chain_of_thought: str
+    answer: int
+
+
+modes = [instructor.Mode.TOOLS, instructor.Mode.TOOLS_STRICT]
+
+
+async def main():
+    uuid = uuid4()
+    print(f"Running eval with uuid: {uuid}")
```

Comment:
Added this so that it's easier to compare across different modes for a single run